ws-calculator/src/main/java/org/egov/wscalculation/config/WSCalculationConfiguration.java

```diff
@@ -182,6 +182,19 @@
     
     @Value("${notification.url}")
     private String notificationUrl;
+
+	@Value("${ws.shortener.url}")
```

Comment:
why use `ws` here? is the `shortener` url specific to `ws`? if not then this should be something `egov.shortener.service.url`